core/src/luigi-config/communication.js

```diff
@@ -39,7 +40,7 @@ export const communication = {
     },
     'busola.setWindowTitle': ({ title }) => {
       const luigiConfig = Luigi.getConfig();
-      luigiConfig.settings.header.title = `Kyma - ${title}`;
+      luigiConfig.settings.header.title = `${config.domain} | ${title}`;
```

Comment:
Maybe it would make more sense to switch places here?
if the domain name would be long, then no one will see the title, and titles are usually short so it shouldn't be a problem the other way around.

Reply:
I've looked around and the patterns are mixed. Let's go for the Suggestion from @dariadomagala ! 

![image](https://user-images.githubusercontent.com/478866/115678660-0915c700-a352-11eb-8de6-945243dd52fb.png)

![image](https://user-images.githubusercontent.com/478866/115677765-1da58f80-a351-11eb-88ab-02685de8879a.png)

Reply:
Can we separate the Window tile from the Luigi title? Having the Application name in front there looks strange.